decidim-sortitions/spec/shared/manage_sortitions_examples.rb

```diff
@@ -0,0 +1,116 @@
+      end
+    end
+
+    it "Requires a random number" do
+      within "form" do
+        expect(page).to have_content(/Result of die roll/i)
+      end
+    end
+
+    it "Requires the number of proposals to select" do
+      within "form" do
+        expect(page).to have_content(/Number of proposals to be selected/i)
+      end
+    end
+
+    it "Requires the proposals component" do
+      within "form" do
+        expect(page).to have_content(/Proposals set/i)
+      end
+    end
+
+    it "Requires the witnesses" do
+      within "form" do
+        expect(page).to have_content(/Witnesses/i)
+      end
+    end
+
+    it "Requires additional information" do
+      within "form" do
+        expect(page).to have_content(/Sortition information/i)
+      end
+    end
+
+    context "when creates a sortition" do
+      let(:sortition_dice) { ::Faker::Number.between(1, 6) }
+      let(:sortition_target_items) { ::Faker::Number.between(1, 10) }
+
+      it "shows the sortition details" do
+        within ".new_sortition" do
+          fill_in :sortition_dice, with: sortition_dice
+          fill_in :sortition_target_items, with: sortition_target_items
+          select translated(proposal_component.name), from: :sortition_decidim_proposals_component_id
+          fill_in_i18n_editor(
+            :sortition_witnesses,
+            "#sortition-witnesses-tabs",
+            en: "Witnesses",
+            es: "Testigos",
+            ca: "Testimonis"
+          )
+          fill_in_i18n_editor(
+            :sortition_additional_info,
+            "#sortition-additional_info-tabs",
+            en: "additional info",
+            es: "Información adicional",
+            ca: "Informació adicional"
+          )
+
+          fill_in_i18n(
+            :sortition_title,
+            "#sortition-title-tabs",
+            en: "Title",
+            es: "Título",
+            ca: "Títol"
+          )
+
+          accept_alert { find("*[type=submit]").click }
+        end
+
+        expect(page).to have_admin_callout("successfully")
+        expect(page).to have_content(/Title/i)
+
+        proposal = Decidim::Sortitions::Sortition.last
+        within ".sortition" do
+          expect(page).to have_content(/Draw time/i)
+          expect(page).to have_content(/Dice/i)
+          expect(page).to have_content(/Items to select/i)
+          expect(page).to have_content(/Category/i)
+          expect(page).to have_content(/All categories/i)
+          expect(page).to have_content(/Proposals component/i)
+          expect(page).to have_content(translated(proposal_component.name))
+          expect(page).to have_content(/Seed/i)
+          expect(page).to have_content(proposal.seed)
+        end
+
+        within ".proposals" do
+          expect(page).to have_content(/Proposals selected for draw/i)
+          proposal.selected_proposals.each do |p|
+            expect(page).to have_content(translated(p.title))
```

Comment:
Looks like this line isn't being hit by the test suite, which makes me think that `selected_proposals` is an empty collection!